.github/workflows/linux_android_arm64.yml

```diff
@@ -0,0 +1,178 @@
+name: Linux Android ARM64 (Ubuntu 20.04, Python 3.11)
+on:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+#  pull_request:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#  push:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#    branches:
+#      - master
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}-linux-android-arm64
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    # TODO: remove. Temporary measure to prevent the workflow from scheduling on forks.
```

Comment:
what are the long term solution for future?

Reply:
Once we have a sufficient # of runners/allowed parallel pipelines, these workflows will be not on the `schedule` trigger, but on a `pull_request` trigger to validate PRs. 
Now, they are enabled only on a nightly basis to check that they run correctly.